exampleSite/content/encounters/uwu.md

```diff
@@ -0,0 +1,14 @@
+---
+title: The Weapon's Refrain
+card_image: /theme-assets/gaia-test.png
+tier_image: /theme-assets/gaia-test.png
+encounter_category: ultimate
+tier_name: Ultimate
+series_name: Ultimate
+weight: 2
+tier_weight: 1
+coming_soon: false
```

Comment:
Do we want to also have a `spoiler` boolean so we could keep something marked as a spoiler for a few weeks after a guide launch? That way, `coming_soon` just adds the text, but `spoiler` enables the css blur. Just me wondering aloud at 2am.

Reply:
I can set up a `spoiler` boolean to create a blur effect. I think there is a CSS way to make it so.